doc/source/management/audit_log_plugin.rst

```diff
@@ -141,18 +141,229 @@ Ability to stream the audit log to `syslog <http://www.syslog.org/>`_ was implem
 
    Variables: :variable:`audit_log_strategy`, :variable:`audit_log_buffer_size`, :variable:`audit_log_rotate_on_size`, :variable:`audit_log_rotations` have effect only with ``FILE`` handler. 
 
+.. _filtering_by_user:
+
+Filtering by user
+=================
+
+In :rn:`5.6.32-78.0` |Percona Server| has implemented filtering by user. This
+was implemented by adding two new global variables:
+:variable:`audit_log_include_accounts` and
+:variable:`audit_log_exclude_accounts` to specify which user accounts should be
+included or excluded from audit logging. 
+
+.. warning:: 
+
+  Only one of these variables can contain the list of users to be either
+  included or exclude, while other needs to be ``NULL``. If one of the
+  variables is set to be not ``NULL`` (contains a list of users), the attempt
+  to set another one will fail. Empty string means an empty list.
+
+.. note::
+
+  Change of :variable:`audit_log_include_accounts` and
+  :variable:`audit_log_exclude_accounts` does not apply to live connections.
+
+Example
+-------
+
+Following example shows adding users who will be monitored: 
+
+.. code-block:: mysql
+
+  mysql> SET GLOBAL audit_log_include_accounts = 'user1@localhost,root@localhost';
+  Query OK, 0 rows affected (0.00 sec)
+
+If you you try to add users to both include and exclude lists server will show
+you the following error:
+
+.. code-block:: mysql
+
+  mysql> SET GLOBAL audit_log_exclude_accounts = 'user1@localhost,root@localhost';
+  ERROR 1231 (42000): Variable 'audit_log_exclude_accounts' can't be set to the value of 'user1@localhost,root@localhost'
+
+You can add users to a different list by first setting the value of list they
```

Comment:
To switch from filtering by included user list to the excluded one or back, first set the currently active filtering variable to NULL: